libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -37,7 +37,7 @@ import { DaffCartErrorType } from '../../reducers/cart-error-type.enum';
 @Injectable({
   providedIn: 'root'
 })
-export class DaffCartFacade implements DaffStoreFacade<Action> {
+export class DaffCartFacade<T extends DaffCart> implements DaffStoreFacade<Action> {
   loading$: Observable<boolean>;
   cart$: Observable<DaffCart>;
```

Comment:
Should this be `cart$: Observable<T>` ? That would probably mean that you do need to pass the generic into the selector, at least for this one.

Reply:
Yea. I made the selectors that don't need to be generic not generic too.